internal/gengapic/custom_operation.go

```diff
@@ -66,12 +68,42 @@ func (g *generator) customOpPointerType() (string, error) {
 // customOpInit builds a string containing the Go code for initializing the
 // operation wrapper type with the Go identifier for a variable that is the
 // proto-defined operation type.
-func (g *generator) customOpInit(h, p string) string {
+func (g *generator) customOpInit(p, r, o string, req *descriptor.DescriptorProto, s *descriptor.ServiceDescriptorProto) {
```

Comment:
nit: given p,r,o are all just strings it may be beneficial to give them better names for readability. Hard to know what they are used for without a stronger type name.

Reply:
Good point, thanks for keeping me from being lazy :) Done.